modules/rdg/src/userobjects/ElementLoopUserObject.C

```diff
@@ -185,10 +185,31 @@ ElementLoopUserObject::onInternalSide(const Elem * elem, unsigned int side)
 }
 
 void
-ElementLoopUserObject::onInterface(const Elem * /*elem*/,
-                                   unsigned int /*side*/,
-                                   BoundaryID /*bnd_id*/)
+ElementLoopUserObject::onInterface(const Elem * elem, unsigned int side, BoundaryID /*bnd_id*/)
 {
+  _current_elem = elem;
+  // Pointer to the neighbor we are currently working on.
+  _current_neighbor = elem->neighbor(side);
+
+  // Get the global id of the element and the neighbor
+  const dof_id_type elem_id = elem->id();
+  const dof_id_type neighbor_id = _current_neighbor->id();
+
+  // TODO: add if-statement to check if this needs to be executed
+  if ((_current_neighbor->active() && (_current_neighbor->level() == elem->level()) &&
+       (elem_id < neighbor_id)) ||
+      (_current_neighbor->level() < elem->level()))
```

Comment:
Internal sides are by definition hit twice. Once from an element and second time form a neighbor.  However, we want to compute things on them just once, therefore `elem_id<neighbor_id`. And this is only valid when element and its neighbor are at the same level of refinement.  

This should be fine, we have this at other places in MOOSE.